.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,28 @@
+name: Docker Image CI
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+
+  build_and_push_latest:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Login to Docker Hub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+    - name: Build docker-compose latest
+      run: GITHUB_API_TOKEN='${{ secrets.GITHUB_TOKEN }}' docker-compose build
+    - name: Push latest image
+      run: GITHUB_API_TOKEN='${{ secrets.GITHUB_TOKEN }}' docker-compose build  
+    - name: Build docker-compose tag
+      run: GITHUB_API_TOKEN='${{ secrets.GITHUB_TOKEN }}' GROCY_IMAGE_TAG="$(git describe --abbrev=0 --tags)" docker-compose build
```

Comment:
Optional: an alternative `git` command to retrieve the tag name for the current commit is:

```
git tag --contains
```